src/components/content/conditions/patient-conditions-filters.tsx

```diff
@@ -1,20 +1,66 @@
 import { useState } from "react";
+import {
+  FilterChangeEvent,
+  FilterItem,
+} from "@/components/core/filter-bar/filter-bar-types";
 import { ConditionModel } from "@/fhir/models";
-import { cloneDeep, merge } from "@/utils/nodash";
+import { compact, isArray, uniq } from "@/utils/nodash";
 
 export type FilterCollection = "patient" | "other";
 
 export type Filters = {
-  showHistoric: boolean;
+  patient: FilterChangeEvent;
+  other: FilterChangeEvent;
+};
+
+const DEFAULT_FILTERS: Filters = {
+  patient: {
+    displayStatus: {
+      key: "displayStatus",
+      selected: ["Active", "Pending", "Unknown"],
+      type: "checkbox",
+    },
+  },
+  other: {
+    displayStatus: {
+      key: "displayStatus",
+      selected: ["Active", "Pending", "Unknown"],
+      type: "checkbox",
+    },
+  },
 };
 
 export function useConditionFilters(collection: FilterCollection) {
   const [filters, setFilters] = useState<Filters>({
-    showHistoric: false,
+    ...DEFAULT_FILTERS,
   });
 
   function updateFilters(newFilters: Partial<Filters>) {
-    setFilters(merge(cloneDeep(filters), newFilters));
+    setFilters(() => ({
+      ...filters,
+      [collection]: {
+        ...newFilters,
+      },
+    }));
+  }
```

Comment:
Not a blocker, can leave this as-is. This looks like a good candidate for set
```suggestion
    setFilters(set(collection, newFilters, filters));
```
where set is from lodash fp _(non-fp set is mutative)_,`import { set } from "@/utils/nodash/fp"`.